doc/api/nmo-activetasks.md

```diff
@@ -0,0 +1,15 @@
+nmo-activetasks(3) -- activetasks
+==============================
+
+## SYNOPSIS
+
+    nmo.commands.activetasks.getActiveTask(url, filter)
```

Comment:
just export `getActiveTask` as default (example https://github.com/apache/couchdb-nmo/blob/master/src/isonline.js#L54) and you can change the api consistent to the other existing:
```
nmo.commands.activetasks(url, filter)
```